components/cstudio-contextual-nav/wcm-site-dropdown-mods/wcm-root-folder.js

```diff
@@ -1374,9 +1374,10 @@ treeNode.getHtml = function() {
                                 success: function (treeData) {
                                     if (currentUri == treeData.item.uri) {
                                         var style = "",
-                                            cont = paramCont ? paramCont : 0;
-                                        YDom.get(curNode.labelElId) ? YDom.get(curNode.labelElId).innerHTML =
-                                            (treeData.item.internalName != "" ? treeData.item.internalName : treeData.item.name) : null;
+                                            cont = paramCont ? paramCont : 0,
+                                            currentInternalName = (treeData.item.internalName != "" ? treeData.item.internalName  : treeData.item.name);
+                                        currentInternalName = treeData.item.isNew ? currentInternalName + " *" : currentInternalName;
```

Comment:
Can you please take a look at the code and see where all these decisions are made.  I feel like we need on place in the code that should be creating icons, setting labels etc....